library/src/blas_ex/rocblas_gemm_ex.hpp

```diff
@@ -351,103 +600,22 @@ rocblas_status gemm_ex_handle_transpose(rocblas_handle    handle,
-    }
-    else if((trans_a == rocblas_operation_transpose
-             || trans_a == rocblas_operation_conjugate_transpose)
-            && (trans_b == rocblas_operation_transpose
-                || trans_b == rocblas_operation_conjugate_transpose))
-    {
-        t_status = tensile_Cijk_Alik_Bjlk_B<Ti, To, Tc>(static_cast<To*>(d),
-                                                        static_cast<const To*>(c_in),
-                                                        static_cast<const Ti*>(a),
-                                                        static_cast<const Ti*>(b),
-                                                        alpha,
-                                                        beta,
-                                                        static_cast<unsigned int>(ldd),
-                                                        stride_d,
-                                                        static_cast<unsigned int>(ldi),
-                                                        stride_i,
-                                                        static_cast<unsigned int>(lda),
-                                                        stride_a,
-                                                        static_cast<unsigned int>(ldb),
-                                                        stride_b,
-                                                        static_cast<unsigned int>(m),
-                                                        static_cast<unsigned int>(n),
-                                                        static_cast<unsigned int>(batch_count),
-                                                        static_cast<unsigned int>(k),
-                                                        handle->rocblas_stream);
-    }
-    else
-    {
-        t_status = tensileStatusFailure;
-    }
-
-    rb_status
-        = t_status == tensileStatusSuccess ? rocblas_status_success : rocblas_status_internal_error;
+    t_status = call_tensile_ex<Ti,To,Tc>(static_cast<To*>(d),
+                                         static_cast<const To*>(c_in),
+                                         static_cast<const Ti*>(a),
+                                         static_cast<const Ti*>(b),
+                                         alpha, beta,
+                                         static_cast<unsigned int>(ldd), stride_d,
+                                         static_cast<unsigned int>(ldi), stride_i,
+                                         static_cast<unsigned int>(lda), stride_a,
+                                         static_cast<unsigned int>(ldb), stride_b,
+                                         static_cast<unsigned int>(m),
+                                         static_cast<unsigned int>(n),
+                                         static_cast<unsigned int>(batch_count),
+                                         static_cast<unsigned int>(k),
```

Comment:
`unsigned(...)` can be used instead of `static_cast<unsigned int>(...)` for brevity.

Reply:
understand I will correct it on third commit "update complex change".